### PR TITLE
Update to 3.2.8

### PR DESCRIPTION
**Pull request by pvuorela**
@mlehtima @Tomin1

### CODE REVIEW

**Review by mlehtima — APPROVED** [review]

File: rpm/gmime.spec [hunk 1]
```diff
@@ -39,25 +34,16 @@ Internet Mail Extension (MIME). The devel-package contains header files
...
-%patch2 -p1
-%__cp $RPM_SOURCE_DIR/gpgme.m4 m4/
 
 %build
-AUTOGEN_SUBDIR_MODE=1 %autogen || true
-
+./autogen.sh
 %configure  \
     --disable-gtk-doc \
     --enable-cryptography=no \
```
Comment (saukko) [inline rpm/gmime.spec]: I wonder if this is wrong based on https://github.com/sailfishos-mirror/gmime/blob/6bff5288f9c108160dd56dd20a9c6608aa55f791/configure.ac

File: rpm/gmime.spec [hunk 2]
```diff
@@ -39,25 +34,16 @@ Internet Mail Extension (MIME). The devel-package contains header files
...
-# Add-automake-1.16.1-support.patch
-%patch2 -p1
-%__cp $RPM_SOURCE_DIR/gpgme.m4 m4/
 
 %build
-AUTOGEN_SUBDIR_MODE=1 %autogen || true
-
+./autogen.sh
 %configure  \
     --disable-gtk-doc \
```
Comment (saukko) [inline rpm/gmime.spec:40]: Extra poins for making explicit disables/enables for the current build configs: https://github.com/sailfishos-mirror/gmime/blob/6bff5288f9c108160dd56dd20a9c6608aa55f791/configure.ac#L679

File: rpm/gmime.spec [hunk 3]
```diff
@@ -3,16 +3,12 @@ Name:       gmime
 %define keepstatic 1
```
Comment (saukko) [inline rpm/gmime.spec]: Hmmp... I wonder if we use this anywhere, would drop if not.

**Review by saukko — APPROVED** [review]

File: rpm/gmime.spec (hunk 1 above)
Comment (pvuorela) [reply rpm/gmime.spec]: Well spotted, the option name got changed a bit during the versions.

File: rpm/gmime.spec (hunk 3 above)
Comment (pvuorela) [reply rpm/gmime.spec]: Defined whether the .a file gets installed, but let's actually get rid of both. We haven't really used those anyway.

File: rpm/gmime.spec (hunk 2 above)
Comment (pvuorela) [reply rpm/gmime.spec:40]: Explicitly disabled the introspection for starters.

**Review by mlehtima — APPROVED** [review]

**Pull request merged**